invite-to-slack/README.md

```diff
@@ -1,2 +1,11 @@
 # Invite Requests
 This directory contains the script we are using in order to receive invite requests through Slack itself. This make the invitation management much much easier for admins to manage 💃
```

Comment:
Let's mention that the invites requests are done through a google form via forms.google.com.